src/Http/Controllers/WebhookController.php

```diff
@@ -199,23 +200,6 @@ protected function handleInvoicePaymentActionRequired(array $payload)
         return $this->successMethod();
     }
 
-    /**
-     * Get the billable entity instance by Stripe ID.
-     *
-     * @param  string|null  $stripeId
-     * @return \Laravel\Cashier\Billable|null
-     */
-    protected function getUserByStripeId($stripeId)
-    {
-        if ($stripeId === null) {
-            return;
-        }
-
-        $model = config('cashier.model');
-
-        return (new $model)->where('stripe_id', $stripeId)->first();
-    }
-
```

Comment:
So, like I said on Discord: it's probably best to keep this method and do the Cashier call inside of here. That way there's no BC break and people can easily overwrite the method if they need to.